packages/@o3r/core/schematics/rule-factories/store/index.ts

```diff
@@ -65,13 +65,12 @@ export function updateStore(options: { projectName?: string | undefined; working
   const updatePackageJson: Rule = (tree: Tree, context: SchematicContext) => {
     const type = projectType === 'library' ? NodeDependencyType.Peer : NodeDependencyType.Default;
 
-    const appDeps = [ngrxEffectsDep, ngrxRouterStoreDevToolDep, fastDeepEqualDep];
+    const appDeps = [ngrxEffectsDep, ngrxRouterStore, ngrxRouterStoreDevToolDep, fastDeepEqualDep];
```

Comment:
no, because `@o3r/application` is using `@o3r/routing` in `/packages/@o3r/application/src/devkit/application-devtools.service.ts`. And `@o3r/routing` needs `@ngrx/router-store`. I suppose that before, by chance, some other package was bringing `@ngrx/router-store`

Reply:
I think the issue is that `@o3r/routing` should be a peer optional in `@o3r/application` as it's used only for the devtools that is not mandatory

Reply:
As the import from @o3r/routing is in the code, yarn pnp makes a static analyze of the code going further and analyzing  also the code from o3r/routing module, which uses an import from '@ngrx/router-store'. IMO changing the dep in optional will not solve the issue. Also the service OtterApplicationDevtools is exported in public-api so I don't think it's optional.